src/logic/__test__/calculator-operator.test.js

```diff
@@ -0,0 +1,56 @@
+// import { render, screen } from '@testing-library/react';
+import calculator from '../calculate';
+import operate from '../operate';
+
+/* CALCULATE.JS  */
+describe('Calculator.js Testing ', () => {
+  test('Test for button name as AC', () => {
+    const obj = {
+      total: 5,
+      next: 2,
+      operation: '',
+    };
+    const buttonName = 'AC';
+
+    const response = calculator(obj, buttonName);
+
+    expect(response.total).toBe(null);
+  });
+
+  test('Test for button name as = ', () => {
+    const obj = {
+      total: 5,
+      next: 2,
+      operation: '+',
+    };
+    const buttonName = '=';
+
+    const response = calculator(obj, buttonName);
+
+    expect(response.total).toBe('7');
+    expect(response.next).toBe(null);
+  });
+});
```

Comment:
- [ ] Well done for writing tests for the `calculate.js`. However, more tests are required. Remember we are writing unit tests and every operation in the calculate.js needs to be tested. Please include missing tests.